tests/foreman/ui/test_docker.py

```diff
@@ -1279,6 +1280,34 @@ def test_positive_delete(self):
                     self.assertIsNone(
                         self.container.search(compute_resource.name, name))
 
+    # BZ 1266842 is a private bug, so we cannot fetch information about it
+    @stubbed('Unstub when BZ1266842 is fixed')
```

Comment:
Yeah, we have bunch of test cases that are using that approach which is definitely not good and should be improved